cypress/e2e/pages/governance/create/index.spec.ts

```diff
@@ -0,0 +1,593 @@
+        "https://www.reddit.com/r/defiblockchain/"
+      );
+      cy.findByTestId(
+        "Governance.Create.GettingStarted.Content.ReadHere"
+      ).should(
+        "have.attr",
+        "href",
+        "https://github.com/DeFiCh/dfips/blob/master/README.md"
+      );
+    });
+
+    it("should have step 1 header and description", () => {
+      cy.findByTestId("Governance.Create.Step1").should("be.visible");
+
+      cy.findByTestId("Governance.Create.Step1").within(() => {
+        cy.findByTestId("Governance.Create.Step1.Title").should(
+          "have.text",
+          "Step 1: Proposal details"
+        );
+
+        cy.findByTestId("Governance.Create.Step1.Description").should(
+          "have.text",
+          "Enter from GitHub or Reddit the title of the proposal and the type of proposal."
+        );
+      });
+    });
+
+    it("should have step 1 cfp radio group checked and dfip radio unchecked", () => {
+      cy.findByTestId("Governance.Create.Step1.RadioGroup").should(
+        "be.visible"
+      );
+      cy.findByTestId("Governance.Create.Step1.RadioGroup").within(() => {
+        cy.findByTestId("Governance.Create.Step1.RadioGroup.CFP")
+          .should("be.visible")
+          .within(() => {
+            cy.findByTestId(
+              "Governance.Create.Step1.RadioGroup.CFP.Checked"
+            ).should("be.visible");
+          });
+
+        cy.findByTestId("Governance.Create.Step1.RadioGroup.DFIP")
+          .should("be.visible")
+          .within(() => {
+            cy.findByTestId(
+              "Governance.Create.Step1.RadioGroup.DFIP.Unchecked"
+            ).should("be.visible");
+          });
+      });
+    });
+
+    it("should have step 1 dfip radio group checked and cfp radio group unchecked", () => {
+      cy.findByTestId("Governance.Create.Step1.RadioGroup.DFIP").click();
+      // DFIP radio
+      cy.findByTestId("Governance.Create.Step1.RadioGroup.DFIP")
+        .should("be.visible")
+        .within(() => {
+          cy.findByTestId(
+            "Governance.Create.Step1.RadioGroup.DFIP.Checked"
+          ).should("be.visible");
+        });
+      // CFP radio
+      cy.findByTestId("Governance.Create.Step1.RadioGroup.CFP")
+        .should("be.visible")
+        .within(() => {
+          cy.findByTestId(
+            "Governance.Create.Step1.RadioGroup.CFP.Unchecked"
+          ).should("be.visible");
+        });
+    });
+
+    it("should have step 1 `Name of proposal` input for DFIP proposal creation", () => {
+      cy.findByTestId("Governance.Create.Step1.TextArea.NameOfProposal").should(
+        "be.visible"
+      );
+      cy.findByTestId("Governance.Create.Step1.TextArea.NameOfProposal").within(
+        () => {
+          cy.get("textarea").click().type(nameOfProposal);
+          cy.findByTestId(
+            "Governance.Create.Step1.TextArea.NameOfProposal.ClearForm"
+          ).click();
+          cy.findByTestId("Governance.Create.Step1.TextArea.ErrorMsg").should(
```

Comment:
can we also please add here a check, that Review Proposal button is inactive; before clear the form, thanks!

Reply:
updated to add checks for review proposal to remain inactive when a single input field is filled